daily-emailing/d_db_funcs.py

```diff
@@ -57,3 +57,20 @@ def get_daily_subscribers() -> list[str]:
     if data:
         return [subscriber['subscriber_email'] for subscriber in data]
     return []
+
+
+def get_yesterday_links() -> list[str]:
+    """Returns the article links from yesterday."""
```

Comment:
Nice appending the links on to the email incase user is curious about article source